src/intervals/precision.jl

```diff
@@ -77,44 +85,13 @@ Base.float{T}(::Type{Rational{T}}) = typeof(float(one(Rational{T})))
 
 # Use that type for rounding with rationals, e.g. for sqrt:
 
-if VERSION < v"0.5.0-dev+1182"
-
-    function Base.with_rounding{T}(f::Function, ::Type{Rational{T}},
-        rounding_mode::RoundingMode)
-        setrounding(f, float(Rational{T}), rounding_mode)
-    end
-
-else
-    function Base.setrounding{T}(f::Function, ::Type{Rational{T}},
-        rounding_mode::RoundingMode)
-        setrounding(f, float(Rational{T}), rounding_mode)
-    end
+function Base.setrounding{T}(f::Function, ::Type{Rational{T}},
```

Comment:
It's just a way to say that rounding is a no-op for rationals.

Currently in Julia `setrounding(T, RoundDown)` is an error for `T` that are not `<: AbstractFloat`, e.g.

```
julia> setrounding(Int, RoundDown)
```

Since we have `Interval`s of `Rational`s, we need this no-op method for code reusability.